docs/serviceability.md

```diff
@@ -0,0 +1,76 @@
+# Serviceability
+
+The serviceability document aims to help local application-service developers and SRE engineers to access and service the application-service component. This document will help you understand how to access and understand the application-service logs, how to debug an application-service problem and provides a quick summary on the various questions that you might have regarding application-service. 
+
+## Accessing the Logs
+
+### Deployed Locally
+View the application-service controller logs by running the executable manager. Example,
+
+```
+make install
+make build
+./bin/manager
+```
+
+### Deployed on a Local Cluster
+View the application-service controller logs by tailing the manager container log of the controller manager pod. The pod resides in the application-service namespace. Example,
+
+```
+oc logs -f application-service-application-service-controller-manager -c manager -n application-service
+```
+
+### Deployed on a Managed Cluster
+
+To access the application-service controller logs on either the AppStudio Staging or Production clusters, you would need access to CloudWatch service. To learn more about how to request access to CloudWatch and view the application-service logs, refer to the HAS Access Control [documentation](https://docs.google.com/document/d/1cK4XGKpXBEYOKfIqSiHuuCfsfHjElxhG9lrlEozzgVE/edit#heading=h.yxk6h5uvh57d). 
+
+## Understanding the Logs
+Each application-service controller logs their reconcile logic to the manager. The log message format is generally of format 
+
+```
+{"level":"info","ts":"2023-08-31T19:59:21.144Z","msg":"Finished reconcile loop for user-tenant/devfile-sample-go-basic-development-binding-hr9nm","controller":"snapshotenvironmentbinding","controllerGroup":"appstudio.redhat.com","controllerKind":"SnapshotEnvironmentBinding","SnapshotEnvironmentBinding":{"name":"devfile-sample-go-basic-development-binding-hr9nm","namespace":"user-tenant"},"namespace":"user-tenant","name":"devfile-sample-go-basic-development-binding-hr9nm","reconcileID":"d5c8545b-957b-4f1a-b177-84a5d5f0d26c"}
+```
+
+To understand the AppStudio controller logging convention, refer to the Appstudio [ADR](https://github.com/redhat-appstudio/book/blob/main/ADR/0006-log-conventions.md)
+
+## Debugging
+
+- Insert break points at the controller functions to debug unit tests
```

Comment:
_How_ to setup HAS with debugger (e.g. dlv in vscode) would be good in this section